### PR TITLE
stage-main 12.2 release candidate

### PR DESCRIPTION
**Pull request by crayolakat**
# Changes
- https://github.com/MoveOnOrg/Spoke/pull/2010 Send auto opt-out response
- https://github.com/MoveOnOrg/Spoke/pull/2076 CiviCRM integration
- https://github.com/MoveOnOrg/Spoke/pull/2116 Fix CSV upload error messages
- https://github.com/MoveOnOrg/Spoke/pull/2147 Fix Auth0 password reset behavior
- https://github.com/MoveOnOrg/Spoke/pull/2160, https://github.com/MoveOnOrg/Spoke/pull/2162 Bandwidth integration
- https://github.com/MoveOnOrg/Spoke/pull/2168 Fix wrong counts in edge cases
- https://github.com/MoveOnOrg/Spoke/pull/2170 Fix wrong service vendor error links
- https://github.com/MoveOnOrg/Spoke/pull/2172 Fix modal going blank after clicking "Clear Response" button
- https://github.com/MoveOnOrg/Spoke/pull/2179, https://github.com/MoveOnOrg/Spoke/pull/2180, https://github.com/MoveOnOrg/Spoke/pull/2181 Update dependencies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Spoke is an open source text-distribution tool for organizations to mobilize sup
 
 Spoke was created by Saikat Chakrabarti and Sheena Pakanati, and is now maintained by MoveOn.org.
 
-The latest version is [12.1](https://github.com/MoveOnOrg/Spoke/tree/v12.1) (see [release notes](https://github.com/MoveOnOrg/Spoke/blob/main/docs/RELEASE_NOTES.md#v121))
+The latest version is [12.2](https://github.com/MoveOnOrg/Spoke/tree/12.2.0) (see [release notes](https://github.com/MoveOnOrg/Spoke/blob/main/docs/RELEASE_NOTES.md#v122))
```
Comment (Arique1104) [inline README.md:10]: Awesome!

File: docs/HOWTO_INTEGRATE_BANDWIDTH.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Bandwidth Integration
```
Comment (Arique1104) [inline docs/HOWTO_INTEGRATE_BANDWIDTH.md:1]: I appreciate how thorough this documentation is.

File: src/lib/parse_csv.js [hunk 3]
```diff
@@ -3,13 +3,52 @@ import _ from "lodash";
...
+
+export const topLevelUploadFields = {
+  firstName: [
+    "firstname",
+    "firstName",
+    "givenname",
+    "givenName",
+    "f_name",
+    "first",
+    "name"
```
Comment (Arique1104) [inline src/lib/parse_csv.js:15]: It's awesome to see this expanded.  It's going to help with a lot of the CSV upload user experience.

**Review by Arique1104 — APPROVED** [review]
Looks great!  There's a lot of anticipation for this pull request.  The features truly meet the moment.

**Review by ibrand — APPROVED** [review]
🎉

**Pull request merged**